lib/Dancer/Tutorial.pod

```diff
@@ -19,7 +19,7 @@ that type of traffic to that specific URL.
 
 This example shows a single HTTP verb "GET" followed by the root URL "/" and an anonymous subroutine which returns
 the string C<"Hello World!">  If you were to run this example, it would display "Hello World!" when you point your
-browser at L<http://localhost:3000>.
+browser at C<http://localhost:3000>.
```

Comment:
I like the other changes, but I am not sure why you changed this one. It's the only thing that is keeping me from giving this a :+1:. Why did you change this link in particular? Thanks!

Reply:
To help user not to click the link by mistake ;-)

Reply:
Because when they do and they will get error page.

Reply:
Thanks for clarifying. :+1: from me!